### ISSUE TITLE
Coupon field needs label other than the placeholder

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13434]
**Describe the bug**
- Automated test detects missing label.
- Placeholders shouldn't be used as labels.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**Expected behavior**
- Associate a label, or use aria-label.
- Use aria-describedby if more instructions are needed. (e.g. enter a promo code here if you have one)
- Screen Reader vocalizes field and required action.

**Screenshots**
![Screen Shot 2021-08-05 at 6 09 42 PM](https://user-images.githubusercontent.com/15113219/128566549-8fea10c9-8d77-4968-9d30-724214eeb316.png)
![Screen Shot 2021-08-05 at 6 14 04 PM](https://user-images.githubusercontent.com/15113219/128566560-04253001-8c95-4314-8104-c8285e1088a5.png)

**Comment by saad-mo** [comment]
Suggested markup change: 

```
<label>Do you like peas?
  <input type="input" name="peas">
</label>

```

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.
![promoCode](https://user-images.githubusercontent.com/45409515/144882397-4bdac24f-d8ee-4499-bb89-a68a72bbc209.png)
